gradle/eclipse.gradle

```diff
@@ -2,11 +2,26 @@ import org.gradle.plugins.ide.eclipse.model.AccessRule
 
 apply plugin: 'eclipse'
 
+tasks.eclipse.dependsOn(cleanEclipse)
```

Comment:
The thing is I don't see how it would ever be inconsistent short of going to the eclipse config and manually screwing it up (which is not a use case we should seriously care about)... This configuration doesn't depend from other settings in non-conventional ways... adding / removing dependencies would force an update on their own...